parser/src/tests/input/testair.airdsl

```diff
@@ -0,0 +1,10 @@
+def TestAir
```

Comment:
Maybe we should just use `.air` as the exension for now to keep it shorter and simpler? Let's also rename this to either `clock` or `system`, since we'll be adding more tests later. So maybe `Def SystemAir` and `system.air`? (or something like this)